app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.where(token: params[:id]).first
+    if user
+      @token = user.token
+    else
+      redirect_to expired_token_path
+    end
+  end
+
+  def expired_token; end
+
+  def create
+    user = User.where(token: params[:token]).first
```

Comment:
For fetching a single record, using `find_by` is more efficient.  Check notes at the bottom of this page - https://www.gotealeaf.com/lessons/a3a6fbb9/assignments/1566